src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -100,6 +110,89 @@ export const AGGREGATIONS = {
     outputType: 'duration',
     isSortable: true,
   },
+  percentile: {
+    parameters: [
+      {
+        kind: 'column',
+        columnTypes: ['duration'],
+        defaultValue: 'transaction.duration',
+        required: true,
+      },
+      {
+        kind: 'value',
+        dataType: 'number',
+        defaultValue: '0.5',
+        required: true,
+      },
+    ],
+    outputType: null,
+    isSortable: true,
+  },
+  error_rate: {
+    parameters: [],
+    outputType: 'number',
+    isSortable: true,
+  },
+  apdex: {
+    parameters: [
+      {
+        kind: 'column',
+        columnTypes: ['duration'],
+        defaultValue: 'transaction.duration',
+        required: true,
+      },
+      {
+        kind: 'value',
+        dataType: 'number',
+        defaultValue: '300',
+        required: true,
+      },
+    ],
+    outputType: 'number',
+    isSortable: true,
+  },
+  impact: {
+    parameters: [
+      {
+        kind: 'column',
+        columnTypes: ['duration'],
+        defaultValue: 'transaction.duration',
+        required: true,
+      },
+      {
+        kind: 'value',
+        dataType: 'number',
+        defaultValue: '300',
+        required: true,
+      },
+    ],
+    outputType: 'number',
+    isSortable: true,
+  },
+  rps: {
+    parameters: [
```

Comment:
I don't understand fully how these are used throughout the frontend code, but RPM and RPS should not have a parameter that users can toggle. This value should only be used in the timeseries queries, to configure how many results are returned. If a user specifies a value for these in the main table results, it will mess up all the results.

Reply:
These definitions are used to build the input sets that we show users. If these are parameterless to the user, I'll remove these parameters.